platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java

```diff
@@ -94,7 +101,7 @@ public void setUserName(@Nullable String userName) {
    * @param password the password for the email server
    */
   public void setPassword(@Nullable String password) {
-    this.password = password;
+    this.password = encryptionService.decryptValue(password);
```

Comment:
❓ Shouldn’t the setter encrypt the password and then where ever it’s used decrypt it?

Reply:
this idea was expressed on the last pr - #6231. I think the consensus was that it was worth exploring, but not worth blocking the pr.